### PR TITLE
Table 1a: added data fetching

### PR DESCRIPTION
**Pull request by minotogna**

### CODE REVIEW

File: webapp/app/assessment/components/dataTable/table/cell/cellOdp.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { useSelector } from 'react-redux'
+
+import { isPrintingMode } from '@webapp/app/assessment/components/print/printAssessment'
+import { formatNumber } from '@common/bignumberUtils'
+// import { acceptNextDecimal } from '@webapp/utils/numberInput'
```
Comment (sorja) [inline webapp/app/assessment/components/dataTable/table/cell/cellOdp.js]: Remove comments?

Comment (minotogna) [reply webapp/app/assessment/components/dataTable/table/cell/cellOdp.js]: It's a work in progress... Updates are not working, and I need that code. Commented out just for eslint

File: webapp/app/assessment/components/dataTable/table/row.js [hunk 2]
```diff
@@ -26,16 +28,16 @@ const Row = props => {
     console.error('Missing renderer for table row', renderer)
```
Comment (sorja) [inline webapp/app/assessment/components/dataTable/table/row.js]: do we need this

File: webapp/app/assessment/components/section/assessmentSectionView.js [hunk 3]
```diff
@@ -27,33 +26,33 @@ const AssessmentSectionView = () => {
...
   const dispatch = useDispatch()
   const countryIso = useCountryIso()
   const i18n = useI18n()
   const disabled = useSelector(FraState.isSectionEditDisabled(sectionName))
 
-  const dataEmpties = useSelector(state =>
-    tableNames.map(tableName => AssessmentState.isSectionDataEmpty(assessmentType, sectionName, tableName)(state))
-  )
+  // const dataEmpties = useSelector(state =>
+  //   tableNames.map(tableName => AssessmentState.isSectionDataEmpty(assessmentType, sectionName, tableName)(state))
```
Comment (sorja) [inline webapp/app/assessment/components/section/assessmentSectionView.js]: remove comments

File: webapp/app/assessment/fra/sections/extentOfForest/extentOfForestValidatorState.js [hunk 4]
```diff
@@ -28,7 +28,7 @@ export const areasNotExceedingTotalLandAreaValidator = datum => state => {
   return greaterThanOrEqualTo(otherLand, 0)
 }
 
-//==== a
+//= === a
```
Comment (sorja) [inline webapp/app/assessment/fra/sections/extentOfForest/extentOfForestValidatorState.js]: ?

Comment (sorja) [reply webapp/app/assessment/fra/sections/extentOfForest/extentOfForestValidatorState.js]: Maybe prettier or eslint does this, but it is a bit unconsistent

File: webapp/app/assessment/components/section/assessmentSectionView.js (hunk 3 above)
Comment (minotogna) [reply webapp/app/assessment/components/section/assessmentSectionView.js]: yes

File: webapp/app/assessment/fra/sections/extentOfForest/extentOfForestValidatorState.js (hunk 4 above)
Comment (minotogna) [reply webapp/app/assessment/fra/sections/extentOfForest/extentOfForestValidatorState.js]: removed

File: webapp/app/assessment/components/dataTable/table/row.js (hunk 2 above)
Comment (minotogna) [reply webapp/app/assessment/components/dataTable/table/row.js]: nope. removed

**Pull request merged**